content/intro-to-storybook/react/it/simple-component.md

```diff
@@ -1,28 +1,28 @@
 ---
-title: 'Build a simple component'
-tocTitle: 'Simple component'
-description: 'Build a simple component in isolation'
+title: 'Costruisci un componente semplice'
+tocTitle: 'Componente semplice'
+description: 'Costruisci un componente semplice in isolamento'
 commit: '4355037'
 ---
 
-We’ll build our UI following a [Component-Driven Development](https://www.componentdriven.org/) (CDD) methodology. It’s a process that builds UIs from the “bottom-up”, starting with components and ending with screens. CDD helps you scale the amount of complexity you’re faced with as you build out the UI.
+Costruiremo la nostra UI seguendo una metodologia [Component-Driven Development](https://www.componentdriven.org/) (CDD). È un processo che costruisce le UI dal "basso verso l'alto", partendo dai componenti e terminando con gli schermi. Il CDD ti aiuta a gestire la quantità di complessità a cui sei sottoposto mentre costruisci la UI.
 
 ## Task
 
-![Task component in three states](/intro-to-storybook/task-states-learnstorybook.png)
+![Componente Task in tre stati](/intro-to-storybook/task-states-learnstorybook.png)
 
-`Task` is the core component of our app. Each task displays slightly differently depending on exactly what state it’s in. We display a checked (or unchecked) checkbox, some information about the task, and a “pin” button, allowing us to move tasks up and down the list. Putting this together, we’ll need these props:
+`Task` è il componente principale della nostra app. Ogni task viene visualizzato leggermente diversamente a seconda dello stato in cui si trova. Visualizziamo una casella di controllo selezionata (o deselezionata), alcune informazioni sul task e un pulsante "pin", che ci permette di spostare i compiti su e giù nella lista. Mettendo insieme tutto ciò, avremo bisogno di queste props:
```

Comment:
**i compiti** it can be changed with **i task** to as in previous parts it's already used the English version.